src/test/java/org/opensearch/neuralsearch/processor/chunker/DelimiterChunkerTests.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.neuralsearch.processor.chunker;
+
+import org.junit.Assert;
+import org.opensearch.test.OpenSearchTestCase;
+
+import java.util.List;
+import java.util.Map;
+
+import static junit.framework.TestCase.assertEquals;
+import static org.junit.Assert.assertThrows;
+import static org.opensearch.neuralsearch.processor.chunker.DelimiterChunker.DELIMITER_FIELD;
+import static org.opensearch.neuralsearch.processor.chunker.DelimiterChunker.MAX_CHUNK_LIMIT_FIELD;
+
+public class DelimiterChunkerTests extends OpenSearchTestCase {
+
+    public void testChunkerWithNoDelimiterField() {
+        DelimiterChunker chunker = new DelimiterChunker();
+        String content = "a\nb\nc\nd";
+        Map<String, Object> inputParameters = Map.of("", "");
+        Exception exception = assertThrows(IllegalArgumentException.class, () -> chunker.validateParameters(inputParameters));
+        Assert.assertEquals("You must contain field: " + DELIMITER_FIELD + " in your parameter.", exception.getMessage());
+    }
+
+    public void testChunkerWithWrongLimitFieldList() {
+        DelimiterChunker chunker = new DelimiterChunker();
+        String content = "a\nb\nc\nd";
+        Map<String, Object> inputParameters = Map.of(MAX_CHUNK_LIMIT_FIELD, List.of("-1"), DELIMITER_FIELD, "\n");
```

Comment:
MAX_CHUNK_LIMIT_FIELD can also not be -1. Please add a related unit test

Reply:
I think we need a default value like -1. And it will do nothing. Do you want to limit the field to be greater than 0?

Reply:
Setting default value to -1 is great. But when we validate the parameters, this value always needs to be a positive integer.

Reply:
Done.